# Publication Monitoring - Year 2026


## 02/01/2026 (need reading)

> Marks, M., Israel, U., Dilip, R. et al. CellSAM: a foundation model for cell segmentation. Nat Methods 22, 2585–2593 (2025). https://doi-org.eproxy.lib.hku.hk/10.1038/s41592-025-02879-w [LY **CellSegmentation**]

> Tianyu Liu, Tinglin Huang, Tong Ding, Hao Wu, Peter Humphrey, Sudhir Perincheri, Kurt Schalper, Rex Ying, Hua Xu, James Zou, Faisal Mahmood, Hongyu Zhao. spEMO: Leveraging Multi-Modal Foundation Models for Analyzing Spatial Multi-Omic and Histopathology Data. bioRxiv 2025.01.13.632818; doi: https://doi.org/10.1101/2025.01.13.632818 [LY **LLM, FM, MultiOmics**]

> Blampey, Q., Benkirane, H., Bercovici, N. et al. Novae: a graph-based foundation model for spatial transcriptomics data. Nat Methods 22, 2539–2550 (2025). https://doi.org/10.1038/s41592-025-02899-6 [LY **FM, STGITIII**]

> Xiao, X., Zhang, L., Zhao, H. et al. Inferring spatial single-cell-level interactions through interpreting cell state and niche correlations learned by self-supervised graph transformer. Nat Mach Intell (2025). https://doi-org.eproxy.lib.hku.hk/10.1038/s42256-025-01161-0 [LY **Transformer, CCC**]

> Lin, S., Wang, Z., Cui, Y. et al. Bridging the dimensional gap from planar spatial transcriptomics to 3D cell atlases. Nat Methods (2025). https://doi-org.eproxy.lib.hku.hk/10.1038/s41592-025-02969-9 [LY **Atlas, 3D**]

> Yi Wang, Zihang He, Yunjie Yan. SPOmiAlign: A modality-agnostic framework for robust and scalable spatial multimodal alignment via feature matching. https://doi.org/10.64898/2025.12.19.695434 [LY **MultiModal, ST**]
