# Publication Monitoring - Year 2024

## 02/04/2024
> Add here if you still remember

> Haviv, D., Remšík, J., Gatie, M., Snopkowski, C., Takizawa, M., Pereira, N., ... & Pe’er, D. (2024). The covariance environment defines cellular niches for spatial inference. Nature Biotechnology, 1-12.

## 24/04/2024

> Hou, Wenpin, and Zhicheng Ji. "Assessing GPT-4 for cell type annotation in single-cell RNA-seq analysis." Nature Methods (2024): 1-4.

> Moravec, Z., Zhao, Y., Voogd, R. et al. Discovery of tumor-reactive T cell receptors by massively parallel library synthesis and screening. Nat Biotechnol (2024). https://doi.org/10.1038/s41587-024-02210-6

> Yuan, Qiuyue, and Zhana Duren. "Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data." Nature Biotechnology (2024): 1-11.

> Qu, Rihao, et al. "Gene trajectory inference for single-cell data by optimal transport metrics." Nature Biotechnology (2024): 1-11.

> Zeng, Yuansong, et al. "Deciphering cell types by integrating scATAC-seq data with genome sequences." Nature Computational Science (2024): 1-14.

> Furer, Nili, et al. "Natural and age-related variation in circulating human hematopoietic stem cells." bioRxiv (2023): 2023-11.

> Sakaue, S., Weinand, K., Isaac, S., Dey, K. K., Jagadeesh, K., Kanai, M., ... & Raychaudhuri, S. (2024). Tissue-specific enhancer–gene maps from multimodal single-cell data identify causal disease alleles. Nature Genetics, 1-12.

> Mitra, S., Malik, R., Wong, W., Rahman, A., Hartemink, A. J., Pritykin, Y., ... & Leslie, C. S. (2024). Single-cell multi-ome regression models identify functional and disease-associated enhancers and enable chromatin potential analysis. Nature Genetics, 1-10.

