# Publication Monitoring - Year 2025

## Techniques

> A good platform for AI model service, from video to text, from manual to programmatic access. Use the API with Zotero, so it's directly summarized when you add an article. https://siliconflow.cn/  [Chrissy]

> 天工（Skywork）超级智能体 - 文档PPT表格网页深度生成. https://www.tiangong.cn/ [Tianjie]


## 16/09/2025 (need reading)

> Liu T, Huang T, Ding T, et al. spEMO: Leveraging Multi-Modal Foundation Models for Analyzing Spatial Multi-Omic and Histopathology Data[J]. BioRxiv, 2025: 2025.01. 13.632818. https://www.biorxiv.org/content/10.1101/2025.01.13.632818v3.full  [LY **Multimodal**]

> Sun, C., Zhang, Y. STHD: probabilistic cell typing of single spots in whole transcriptome spatial data with high definition. Genome Biol 26, 213 (2025). https://doi.org/10.1186/s13059-025-03608-4  [LY **Cell type**]

> Yoon H, Jung J, Kim J, et al. Visual Representation Alignment for Multimodal Large Language Models[J]. arXiv preprint arXiv:2509.07979, 2025. https://arxiv.org/abs/2509.07979 [LY **Multimodal**]

> Luo, Bingying, et al. "StereoMM: a graph fusion model for integrating spatial transcriptomic data and pathological images." Briefings in bioinformatics 26.3 (2025): bbaf210. https://academic.oup.com/bib/article/26/3/bbaf210/8142327?login=false  [LY **Multimodal**]


## 25/08/2025 (need reading)
> Andani S, Chen B, Ficek-Pascual J, et al. Histopathology-based protein multiplex generation using deep learning[J]. Nature Machine Intelligence, 2025: 1-16. https://www-nature-com.eproxy.lib.hku.hk/articles/s42256-025-01074-y [LY **Image-Hot/Cold tumor**]

> Boehm K M, Aherne E A, Ellenson L, et al. Multimodal data integration using machine learning improves risk stratification of high-grade serous ovarian cancer[J]. Nature Cancer, 2022, 3(6): 723-733. https://www-nature-com.eproxy.lib.hku.hk/articles/s43018-022-00388-9 [LY **BRCA Risk**]

> Wu E, Bieniosek M, Wu Z, et al. ROSIE: AI generation of multiplex immunofluorescence staining from histopathology images[J]. Nature Communications, 2025, 16(1): 7633. https://www-nature-com.eproxy.lib.hku.hk/articles/s41467-025-62346-0 [LY **Image**]

> Li S, Wang R, Liu S, et al. Finding spatially variable ligand-receptor interactions with functional support from downstream genes[J]. Nature Communications, 2025, 16(1): 7784. https://www.nature.com/articles/s41467-025-62988-0 [LY **CCC**]

> Zhang P, Chen W, Tran T N, et al. Thor: a platform for cell-level investigation of spatial transcriptomics and histology[J]. Nature Communications, 2025, 16(1): 7178. https://www.nature.com/articles/s41467-025-62593-1 [LY **Image**]

> Zhao T, Zhang K, Hollenberg M, et al. SimSpace: a comprehensive in-silico spatial omics data simulation framework[J]. bioRxiv, 2025: 2025.07. 18.665587. https://www.biorxiv.org/content/10.1101/2025.07.18.665587v1 [LY **ST Simulation**]

> Zohora F T, Paliwal D, Flores-Figueroa E, et al. CellNEST reveals cell–cell relay networks using attention mechanisms on spatial transcriptomics[J]. Nature Methods, 2025: 1-15. https://www-nature-com.eproxy.lib.hku.hk/articles/s41592-025-02721-3 [LY **CCC**]


## 01/09/2025 (LY, Fei Chen Lab)
> Gong Q, Cao K, Weir J A, et al. Energy-Regularized Graph Learning for Multiscale Spatial Representation[J]. bioRxiv, 2025: 2025.08. 22.671846.  https://www.biorxiv.org/content/10.1101/2025.08.22.671846v1 [LY **Energy**]

> Kambhampati S, D’Alessio L, Grab F, et al. TissueMosaic enables cross-sample differential analysis of spatial transcriptomics datasets through self-supervised representation learning[C]//International Conference on Research in Computational Molecular Biology. Cham: Springer Nature Switzerland, 2025: 386-389. https://www.biorxiv.org/content/10.1101/2024.11.07.622479v1.full  [LY **Cross-sample**]

> Hu C, Borji M, Marrero G J, et al. Scalable spatial transcriptomics through computational array reconstruction[J]. Nature Biotechnology, 2025: 1-7. https://doi-org.eproxy.lib.hku.hk/10.1038/s41587-025-02612-0  [LY **Scalbility**]


## 27/08/2025 (LY)
> You R, Shen Q, Lin C, et al. Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma[J]. Nature Genetics, 2025: 1-16. https://www-nature-com.eproxy.lib.hku.hk/articles/s41588-025-02253-8 [LY **NPC Image-need**]

> Ferri-Borgogno S, Zhu Y, Sheng J, et al. Spatial transcriptomics depict ligand–receptor cross-talk heterogeneity at the tumor-stroma interface in long-term ovarian cancer survivors[J]. Cancer research, 2023, 83(9): 1503-1516. https://aacrjournals-org.eproxy.lib.hku.hk/cancerres/article/83/9/1503/726052/Spatial-Transcriptomics-Depict-Ligand-Receptor [LY **HGSOC ROI Risk**]

> Nirmal A J, Sorger P K. SCIMAP: a python toolkit for integrated spatial analysis of multiplexed imaging data[J]. Journal of open source software, 2024, 9(97): 6604. https://joss.theoj.org/papers/10.21105/joss.06604 [LY **ROI**]


## 17/09/2025
> SpatialZoomer: multi-scale feature analysis of spatial transcriptomics. https://www.biorxiv.org/content/10.1101/2025.09.08.674870v1 [LY **ROI Patch**]

> Schroeder, A., Loth, M.L., Luo, C. et al. Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE. Nat Methods (2025). https://doi-org.eproxy.lib.hku.hk/10.1038/s41592-025-02770-8 [LY **HE image**]

> Fu, X., Cao, Y., Bian, B. et al. Spatial gene expression at single-cell resolution from histology using deep learning with GHIST. Nat Methods (2025). https://doi-org.eproxy.lib.hku.hk/10.1038/s41592-025-02795-z  [LY **HE image**]

> Chitra U, Dan S, Krienen F, et al. GASTON-Mix: a unified model of spatial gradients and domains using spatial mixture-of-experts[J]. Bioinformatics, 2025, 41(Supplement_1): i523-i532. https://academic.oup.com/bioinformatics/article/41/Supplement_1/i523/8199348 [YL **Gradient**]

> Xia, CR., Cao, ZJ. & Gao, G. DECIPHER for learning disentangled cellular embeddings in large-scale heterogeneous spatial omics data. Nat Commun 16, 7991 (2025). https://doi-org.eproxy.lib.hku.hk/10.1038/s41467-025-63140-8 [James **Large scale**]

> Ning Zhang, Yahui Long, Sibo Xia, Pengzhen Jia, Yujie Jin, Siqi Chen, Jinmiao Chen, Min Li. SpaFoundation: a visual foundation model for spatial transcriptomics. https://doi.org/10.1101/2025.08.07.669202 
 [Tianjie **Foundation**]

> Tianyu Liu, Xiangyu Zhang, Jiecong Lin, Luca Pinello, Rex Ying, Hongyu Zhao. Pre-training Genomic Language Model with Variants for Better Modeling Functional Genomics. https://doi.org/10.1101/2025.02.26.640468  [Shumin **LLM**]


## 27/08/2025
> Wen, B., Wang, C., Li, K. et al. DeepMVP: deep learning models trained on high-quality data accurately predict PTM sites and variant-induced alterations. Nat Methods (2025). https://doi.org/10.1038/s41592-025-02797-x  [Kevin **PTM**]

> Yang, P., Jin, K., Yao, Y. et al. Spatial integration of multi-omics single-cell data with SIMO. Nat Commun 16, 1265 (2025). https://doi.org/10.1038/s41467-025-56523-4  [Chrisy **multi-omics**]

> Han, C., Lin, S., Wang, Z. et al. Reusability report: Exploring the transferability of self-supervised learning models from single-cell to spatial transcriptomics. Nat Mach Intell (2025). https://doi.org/10.1038/s42256-025-01097-5  [BX **SSL**]


## 21/08/2025
> Zhao S, Luo Y, Yang G, et al. SToFM: a Multi-scale Foundation Model for Spatial Transcriptomics[J]. arXiv preprint arXiv:2507.11588, 2025. https://arxiv.org/abs/2507.11588 [TJ **ST Foundation**]

> Zhu Q, Zheng Y, Sang Y, et al. Suica: Learning super-high dimensional sparse implicit neural representations for spatial transcriptomics[J]. arXiv preprint arXiv:2412.01124, 2024. https://icml.cc/virtual/2025/poster/44957 [TJ **ST Sparse**]


## 25/06/2025
> Bakulin A, Levy N, Ergen C, et al. NOLAN: SELF-SUPERVISED FRAMEWORK FOR MAPPING CONTINUOUS TISSUE ORGANIZATION[C]//ICLR 2025 Workshop on Machine Learning for Genomics Explorations. https://openreview.net/forum?id=DzlhvkuFqz [Tianjie **Niches**]

> Peng X, Smithy J W, Yosofvand M, et al. Scalable topic modelling decodes spatial tissue architecture for large-scale multiplexed imaging analysis[J]. Nature Communications, 2025, 16(1): 6619. https://www-nature-com.eproxy.lib.hku.hk/articles/s41467-025-61821-y [LY **Topic**]

> Busarello E, Biancon G, Cimignolo I, et al. Cell Marker Accordion: interpretable single-cell and spatial omics annotation in health and disease[J]. Nature Communications, 2025, 16(1): 5399. https://www.nature.com/articles/s41467-025-60900-4 [Lu **Cell Marker**]

> Dong M, Su D G, Kluger H, et al. SIMVI disentangles intrinsic and spatial-induced cellular states in spatial omics data[J]. Nature Communications, 2025, 16(1): 2990. https://www.nature.com/articles/s41467-025-60900-4 [Lu **Cellular State**]

> Predicting Ligand-Receptor Pairing using Spatial Transcriptomics. [YH **CCC**]


## 25/06/2025

> AlphaGenome: AI for better understanding the genome. https://deepmind.google/discover/blog/alphagenome-ai-for-better-understanding-the-genome/ [SM, **Genome**]

> Zhang J, Tian R, Liu J, et al. A two-front nutrient supply environment fuels small intestinal physiology through differential regulation of nutrient absorption and host defense[J]. Cell, 2024, 187(22): 6251-6271. e20. https://linkinghub.elsevier.com/retrieve/pii/S0092-8674(24)00903-6 [BX, **CCC**]

> Zohora F T, Paliwal D, Flores-Figueroa E, et al. CellNEST reveals cell–cell relay networks using attention mechanisms on spatial transcriptomics[J]. Nature Methods, 2025: 1-15. CellNEST reveals cell–cell relay networks using attention mechanisms on spatial transcriptomi [LY, **CCC**]

> Qi J, Luo Z, Li C Y, et al. Interpretable niche-based cell‒cell communication inference using multi-view graph neural networks[J]. Nature Computational Science, 2025: 1-12. https://www.nature.com/articles/s43588-025-00809-6 [LY, **CCC**]

> Chen W, Zhang P, Tran T N, et al. A visual–omics foundation model to bridge histopathology with spatial transcriptomics[J]. Nature Methods, 2025: 1-15. https://www.nature.com/articles/s41592-025-02707-1 [LY, **Foundation**]

> Zhang J, Wang J, Zhou Q, et al. Spatiotemporally resolved transcriptomics reveals the cellular dynamics of human retinal development[J]. Nature Communications, 2025, 16(1): 2307. https://www.nature.com/articles/s41467-025-57625-9 [LY, **CCC**]

> Wu Z, Boen J, Jindal S, et al. Spatial multi-omics and deep learning reveal fingerprints of immunotherapy response and resistance in hepatocellular carcinoma[J]. bioRxiv, 2025: 2025.06. 11.656869. https://www.biorxiv.org/content/10.1101/2025.06.11.656869v1 [YH, **HCC2states**]

> Scherer M, Singh I, Braun M M, et al. Clonal tracing with somatic epimutations reveals dynamics of blood ageing[J]. Nature, 2025. https://www.nature.com/articles/s41586-025-09041-8 [YH, **Clonal**]

> Xi X, Li J, Jia J, et al. A mechanism-informed deep neural network enables prioritization of regulators that drive cell state transitions[J]. Nature Communications, 2025, 16(1): 1284. https://www.nature.com/articles/s41467-025-56475-9 [YH, **regulators**]
> 
> de Jong G, Memi F, Gracia T, et al. A spatiotemporal cancer cell trajectory underlies glioblastoma heterogeneity[J]. bioRxiv, 2025: 2025.05. 13.653495. https://www.biorxiv.org/content/10.1101/2025.05.13.653495v1 [YH, **glioblastoma**]

> Saraswat M, Rueda-Gensini L, Heinzelmann E, et al. Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics[J]. bioRxiv, 2025: 2025.05. 13.653733. https://www.biorxiv.org/content/10.1101/2025.05.13.653733v1.full.pdf [YH, **glioblastoma**]
> 
> He Y, Fang P, Shan Y, et al. Generalized biological foundation model with unified nucleic acid and protein language[J]. Nature Machine Intelligence, 2025: 1-12. https://www.nature.com/articles/s42256-025-01044-4 [James, **LLM**]


## 28/05/2025
> Zeng, Y., Xie, J., Shangguan, N. et al. CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells. Nat Commun 16, 4679 (2025). https://doi.org/10.1038/s41467-025-59926-5, https://mp.weixin.qq.com/s/WS64nT2a1DofG_pxZDVYzA [LY, **scFoundation model**]

> Troulé K, Petryszak R, Cakir B, et al. CellPhoneDB v5: inferring cell–cell communication from single-cell multiomics data[J]. Nature Protocols, 2025: 1-29. https://www.nature.com/articles/s41596-024-01137-1 [LY, **CCC**]

> Barbadilla-Martínez L, Klaassen N, van Steensel B, et al. Predicting gene expression from DNA sequence using deep learning models[J]. Nature Reviews Genetics, 2025: 1-15. https://www.nature.com/articles/s41576-025-00841-2 [LY, **Sequence model**]

> Tang F, Nian B, Li Y, et al. MambaMIM: Pre-training Mamba with state space token interpolation and its application to medical image segmentation[J]. Medical Image Analysis, 2025: 103606. https://www.sciencedirect.com/science/article/abs/pii/S1361841525001537 [LY, **Multi-scale**]

> Dwivedi V P, Bresson X. A generalization of transformer networks to graphs[J]. arXiv preprint arXiv:2012.09699, 2020. https://arxiv.org/abs/2012.09699 [BX, **Graph Transformer**]

> Feng Y, You H, Zhang Z, et al. Hypergraph neural networks[C]//Proceedings of the AAAI conference on artificial intelligence. 2019, 33(01): 3558-3565. https://ojs.aaai.org/index.php/AAAI/article/view/4235 [BX, **Hypergraph**]

> Jones D C, Elz A E, Hadadianpour A, et al. Proseg: Cell simulation as cell segmentation[J]. bioRxiv, 2024. https://www.nature.com/articles/s41592-025-02697-0 [LY, **CellSegment**]


## 30/04/2025

> Bao X, Bai X, Liu X, et al. Spatially informed graph transformers for spatially resolved transcriptomics[J]. Communications Biology, 2025, 8(1): 574. https://www.nature.com/articles/s42003-025-08015-w [James, **RNA+Image**]

> Jiang J, Liu Y, Qin J, et al. METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics[J]. Nature Communications, 2024, 15(1): 7312. https://www.nature.com/articles/s41467-024-51708-9 [LY, **Morphology**]

> Huang J, Yuan C, Jiang J, et al. MorphLink: Bridging Cell Morphological Behaviors and Molecular Dynamics in Multi-modal Spatial Omics[J]. bioRxiv, 2024: 2024.08. 24.609528. https://www.biorxiv.org/content/10.1101/2024.08.24.609528v1 [LY, **Morphology**]

> Jia G, He P, Dai T, et al. **Spatial immune scoring system predicts hepatocellular carcinoma recurrence**[J]. Nature, 2025: 1-11. https://www.nature.com/articles/s41586-025-08668-x [YH, **No Image**]

> Heidari E, Moorman A, Unyi D, et al. Segger: Fast and accurate cell segmentation of imaging-based spatial transcriptomics data[J]. bioRxiv, 2025: 2025.03. 14.643160. https://www.biorxiv.org/content/10.1101/2025.03.14.643160v1 [YH, **CellSegment**]

> Pei G, Min J, Rajapakshe K I, et al. Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer[J]. Nature, 2025: 1-10. https://www.nature.com/articles/s41586-025-08927-x [RT, **No Image**]
