# Publication Monitoring - Year 2024

## 27/11/2024

> Wu, Y., Zhou, J. Y., Yao, B., Cui, G., Zhao, Y. L., Gao, C. C., ... & Yang, Y. G. (2024).
  STASCAN deciphers fine-resolution cell distribution maps in spatial transcriptomics by deep learning. Genome Biology, 25(1), 278.

> Lederer, A. R., Leonardi, M., Talamanca, L., Bobrovskiy, D. M., Herrera, A., Droin, C., ... & La Manno, G. (2024).
  [Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed
  modulations](https://www.nature.com/articles/s41592-024-02471-8).
  Nature Methods, 1-16.
  
> Lu, Z., Mo, S., Xie, D., Zhai, X., Deng, S., Zhou, K., ... & Hu, Z. (2024).
  [Polyclonal-to-monoclonal transition in colorectal precancerous
  evolution](https://www.nature.com/articles/s41586-024-08133-1).
  Nature, 1-8.

> Galassi, C., Chan, T. A., Vitale, I., & Galluzzi, L. (2024).
  [The hallmarks of cancer immune evasion](https://www.cell.com/cancer-cell/fulltext/S1535-6108(24)00358-1).
  Cancer Cell, 42(11), 1825-1863.]

> Nicol, P. B., Ma, R., Xu, R. J., Moffitt, J. R., & Irizarry, R. A. (2024).
  [Identifying spatially variable genes by projecting to morphologically relevant
  curves](https://www.biorxiv.org/content/10.1101/2024.11.21.624653v1). bioRxiv, 2024-11.

> Guo, W., Li, X., Wang, D., Yan, N., Hu, Q., Yang, F., ... & Gu, J. (2024).
  [scStateDynamics: deciphering the drug-responsive tumor cell state dynamics by modeling single-cell level
  expression changes](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03436-y).
  Genome Biology, 25(1), 1-25.

> Kim, M. C., Gate, R., Lee, D. S., Tolopko, A., Lu, A., Gordon, E., ... & Ye, C. J. (2024).
  [Method of moments framework for differential expression analysis of single-cell RNA sequencing
  data](https://www.cell.com/cell/fulltext/S0092-8674%2824%2901144-9). Cell.
  
> Pan, Xinhai, Alejandro Danies-Lopez, and Xiuwei Zhang.
  "Mapping lineage-resolved scRNA-seq data with spatial transcriptomics using TemSOMap." bioRxiv (2024): 2024-10.


## 05/09/2024

> Zhu, J., Wang, Y., Chang, W. Y., Malewska, A., Napolitano, F., Gahan, J. C., ... & Wang, T. (2023).
  Mapping cellular interactions from spatially resolved transcriptomics data. Nature Methods.

> Pham, D., Tan, X., Balderson, B. et al. Robust mapping of spatiotemporal trajectories and cell–cell interactions in healthy and diseased tissues. Nat Commun 14, 7739 (2023). https://doi.org/10.1038/s41467-023-43120-6

## 05/06/2024

> Tayyebi, Z., Pine, A. R., & Leslie, C. S. (2024). 
  Scalable and unbiased sequence-informed embedding of single-cell ATAC-seq data with CellSpace. Nature Methods, 1-9.

> Mannens, C. C., Hu, L., Lönnerberg, P., ... & Linnarsson, S. (2024). 
  Chromatin accessibility during human first-trimester neurodevelopment. Nature, 1-8.

> Wamsley, B., Bicks, L., Cheng, Y., ... & Geschwind, D. H. (2024). 
  Molecular cascades and cell type–specific signatures in ASD revealed by single-cell genomics. Science, 384(6698), eadh2602.

> Emani, P. S., Liu, J. J., Clarke, D., ... & Gerstein, M. (2024). 
  Single-cell genomics and regulatory networks for 388 human brains. Science, 384(6698), eadi5199.

> Huuki-Myers, L. A., Spangler, A., Eagles, N. J., ... & Maynard, K. R. (2024). 
  A data-driven single-cell and spatial transcriptomic map of the human prefrontal cortex. Science, 384(6698), eadh1938.

## 24/04/2024

> Hou, Wenpin, and Zhicheng Ji. "Assessing GPT-4 for cell type annotation in single-cell RNA-seq analysis." Nature Methods (2024): 1-4.

> Moravec, Z., Zhao, Y., Voogd, R. et al. Discovery of tumor-reactive T cell receptors by massively parallel library synthesis and screening. Nat Biotechnol (2024). https://doi.org/10.1038/s41587-024-02210-6

> Yuan, Qiuyue, and Zhana Duren. "Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data." Nature Biotechnology (2024): 1-11.

> Qu, Rihao, et al. "Gene trajectory inference for single-cell data by optimal transport metrics." Nature Biotechnology (2024): 1-11.

> Zeng, Yuansong, et al. "Deciphering cell types by integrating scATAC-seq data with genome sequences." Nature Computational Science (2024): 1-14.

> Furer, Nili, et al. "Natural and age-related variation in circulating human hematopoietic stem cells." bioRxiv (2023): 2023-11.

> Sakaue, S., Weinand, K., Isaac, S., Dey, K. K., Jagadeesh, K., Kanai, M., ... & Raychaudhuri, S. (2024). Tissue-specific enhancer–gene maps from multimodal single-cell data identify causal disease alleles. Nature Genetics, 1-12.

> Mitra, S., Malik, R., Wong, W., Rahman, A., Hartemink, A. J., Pritykin, Y., ... & Leslie, C. S. (2024). Single-cell multi-ome regression models identify functional and disease-associated enhancers and enable chromatin potential analysis. Nature Genetics, 1-10.

## 02/04/2024

> Add here if you still remember

> Haviv, D., Remšík, J., Gatie, M., Snopkowski, C., Takizawa, M., Pereira, N., ... & Pe’er, D. (2024). The covariance environment defines cellular niches for spatial inference. Nature Biotechnology, 1-12.
