# Publication Monitoring - Year 2025

## Techniques

> A good platform for AI model service, from video to text, from manual to programmatic access. Use the API with Zotero, so it's directly summarized when you add an article. https://siliconflow.cn/  [Chrissy]

> 天工（Skywork）超级智能体 - 文档PPT表格网页深度生成. https://www.tiangong.cn/ [Tianjie]


## 28/05/2025

> Troulé K, Petryszak R, Cakir B, et al. CellPhoneDB v5: inferring cell–cell communication from single-cell multiomics data[J]. Nature Protocols, 2025: 1-29. https://www.nature.com/articles/s41596-024-01137-1 [LY, **CCC**]

> Barbadilla-Martínez L, Klaassen N, van Steensel B, et al. Predicting gene expression from DNA sequence using deep learning models[J]. Nature Reviews Genetics, 2025: 1-15. https://www.nature.com/articles/s41576-025-00841-2 [LY, **Sequence model**]

> Tang F, Nian B, Li Y, et al. MambaMIM: Pre-training Mamba with state space token interpolation and its application to medical image segmentation[J]. Medical Image Analysis, 2025: 103606. https://www.sciencedirect.com/science/article/abs/pii/S1361841525001537 [LY, **Multi-scale**]

> Dwivedi V P, Bresson X. A generalization of transformer networks to graphs[J]. arXiv preprint arXiv:2012.09699, 2020. https://arxiv.org/abs/2012.09699 [BX, **Graph Transformer**]

> Feng Y, You H, Zhang Z, et al. Hypergraph neural networks[C]//Proceedings of the AAAI conference on artificial intelligence. 2019, 33(01): 3558-3565. https://ojs.aaai.org/index.php/AAAI/article/view/4235 [BX, **Hypergraph**]

> Jones D C, Elz A E, Hadadianpour A, et al. Proseg: Cell simulation as cell segmentation[J]. bioRxiv, 2024. https://www.nature.com/articles/s41592-025-02697-0 [LY, **CellSegment**]


## 30/04/2025

> Bao X, Bai X, Liu X, et al. Spatially informed graph transformers for spatially resolved transcriptomics[J]. Communications Biology, 2025, 8(1): 574. https://www.nature.com/articles/s42003-025-08015-w [James, **RNA+Image**]

> Jiang J, Liu Y, Qin J, et al. METI: deep profiling of tumor ecosystems by integrating cell morphology and spatial transcriptomics[J]. Nature Communications, 2024, 15(1): 7312. https://www.nature.com/articles/s41467-024-51708-9 [LY, **Morphology**]

> Huang J, Yuan C, Jiang J, et al. MorphLink: Bridging Cell Morphological Behaviors and Molecular Dynamics in Multi-modal Spatial Omics[J]. bioRxiv, 2024: 2024.08. 24.609528. https://www.biorxiv.org/content/10.1101/2024.08.24.609528v1 [LY, **Morphology**]

> Jia G, He P, Dai T, et al. Spatial immune scoring system predicts hepatocellular carcinoma recurrence[J]. Nature, 2025: 1-11. https://www.nature.com/articles/s41586-025-08668-x [YH, **No Image**]

> Heidari E, Moorman A, Unyi D, et al. Segger: Fast and accurate cell segmentation of imaging-based spatial transcriptomics data[J]. bioRxiv, 2025: 2025.03. 14.643160. https://www.biorxiv.org/content/10.1101/2025.03.14.643160v1 [YH, **CellSegment**]

> Pei G, Min J, Rajapakshe K I, et al. Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer[J]. Nature, 2025: 1-10. https://www.nature.com/articles/s41586-025-08927-x [RT, **No Image**]
